packages/react-components/react-toast/src/state/vanilla/toaster.ts

```diff
@@ -0,0 +1,80 @@
+import { Toast, ToastEventMap, ToastId, ToastOptions } from '../types';
+import { EVENTS } from '../constants';
+
+// TODO convert to closure
+export class Toaster {
+  public visibleToasts: Set<ToastId>;
+  public toasts: Map<ToastId, Toast>;
```

Comment:
Let's handle it in a separate PR, but I don't think that these should be exposed as is.